src/components/PeopleInput.tsx

```diff
@@ -0,0 +1,45 @@
+import debounce from 'lodash.debounce';
```

Comment:
try to fix this linter error
![image](https://github.com/mate-academy/react_autocomplete/assets/91946036/8710bd9e-bf91-440b-be61-fc796a6ae267)